bluebottle/suggestions/views.py

```diff
@@ -29,13 +29,13 @@ def get_queryset(self):
 
 class SuggestionDetail(generics.RetrieveUpdateAPIView):
     queryset = Suggestion.objects.all()
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (AllowAny,)
```

Comment:
It seems you can even view the token in the detail view. So yes, this seems like an security issue.